packages/utils/src/date/index.ts

```diff
@@ -14,10 +14,9 @@ interface ConversionOptions {
  *
  * @see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  */
-function getUTCOffset(timeZone: string): number {
+export function getUTCOffset(timeZone: string): number {
 	// Build localized formats for UTC and the target timezone
 	const formatOptions: Intl.DateTimeFormatOptions = {
-		hourCycle: 'h23',
```

Comment:
Not in the type declaration of `Intl.DateTimeFormatOptions`, and doesn't impact the result.